src/reducers/versions.spec.tsx

```diff
@@ -117,12 +120,48 @@ describe(__filename, () => {
 
       expect(state).toEqual({
         ...initialState,
+        expandedPaths: getParentFolders(version.file.selected_file),
         versionInfo: {
           [version.id]: createInternalVersion(version),
         },
       });
     });
 
+    it('prunes expandedPaths when loading version info', () => {
+      const version = { ...fakeVersion, id: nextUniqueId() };
+      const newVersion = { ...fakeVersion, id: nextUniqueId() };
+      const _getPrunedExpandedPaths = jest.fn(getPrunedExpandedPaths);
+      const _reducer = createReducer({ _getPrunedExpandedPaths });
+
+      const state = _reducer(undefined, actions.loadVersionInfo({ version }));
+      _reducer(state, actions.loadVersionInfo({ version: newVersion }));
+
+      expect(_getPrunedExpandedPaths).toHaveBeenCalledWith(
+        state.expandedPaths,
```

Comment:
I am not very sure to have `state.expandedPaths` (which might be too simple like [] or [ROOT]) in this assertion, but if I predefine `expandedPaths = [path1, path2]`, then I will need to create a fake version with entries and dispatch `expandTree`.